dashboard/app/controllers/levels_controller.rb

```diff
@@ -534,7 +534,7 @@ def embed_level
     # Reference links should be stored as an array.
     if params[:level][:reference_links].is_a? String
       params[:level][:reference_links] = params[:level][:reference_links].split("\r\n")
-      params[:level][:reference_links].delete_if(&:blank?)
+      params[:level][:reference_links].compact_blank!
```

Comment:
I am just curious, is this being forced by a newly enabled lint rule?

Reply:
Yep: [Rails/CompactBlank](https://docs.rubocop.org/rubocop-rails/cops_rails.html#railscompactblank)